themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -0,0 +1,182 @@
+<?php $access = $this->driver->getOverdriveAccess();?>
+<?php if ($access->code == 'od_account_noaccess'): ?>
+  <div class="hold">
+    <div class="alert alert-warning">
+      <?= $this->translate('od_code_contentnotavail') ?>
+    </div>
+  </div>
+<?php elseif ($access->code == 'od_code_login_for_avail'): ?>
+  <p class='alert-info'><strong>
+      <?= $this->transEsc('od_code_login_for_avail') ?>
+    </strong></p>
+<?php else:
+  $od_id = $this->driver->getOverdriveID();
+  $rec_id = $this->driver->getUniqueID();
+  $hold_url = $this->url('overdrive-hold');
+  $avail = $this->driver->getOverdriveAvailability();
+  $checkedOutResult = $this->driver->isCheckedOut();
+  $checkedOut = $checkedOutResult->data;
+  if (!$checkedOut):
+    $hold = $this->driver->isHeld();
+  endif; ?>
+
+  <div class="internetDetails">
+    <div class="odbrand pull-left media-left">
+      <img
+        src="https://developerportaldev.blob.core.windows.net/media/Default/images/newLogos/OverDrive_Logo_42x42_rgb.jpg">
+    </div>
+    <h3>
+      <?= $this->transEsc('Online Access') ?>
+    </h3>
+    <div class="availability">
+      <p>
+        <?php if ($avail->data->copiesAvailable > 0): ?>
+          <span class="text-success">
+            <?= $this->icon('status-available', 'icon-link__icon') ?>
+          </span>
+        <?php else: ?>
+          <span class="text-danger">
+            <?= $this->icon('status-unavailable', 'icon-link__icon') ?>
+          </span>
+        <?php endif; ?>
+
+        <?php if (isset($avail->data->availabilityType) && $avail->data->availabilityType == 'AlwaysAvailable'): ?>
+          <?= $this->transEsc('od_unlimited') ?>
+        <?php else: ?>
+          <?= $avail->data->copiesAvailable . '/' . $avail->data->copiesOwned ?>
+        <?php endif; ?>
+        <?= $this->transEsc('od_copies_available') ?>
+        <?php if ($avail->data->numberOfHolds > 0): ?>
+          <?= '; ' . $avail->data->numberOfHolds . ' ' . ($avail->data->numberOfHolds > 1 ? $this->transEsc('od_people') : $this->transEsc('od_person')) . ' ' . $this->transEsc('od_waiting') ?>
```

Comment:
This looks like an opportunity to use our recently-added support for ICU message strings. Instead of separate od_people/od_person strings, I'd suggest changing this to:

```
<?= '; ' . $this->transEsc('od_waiting', ['holds' => $avail->data->numberOfHolds], null, true) ?>
```

And then you can make the od_waiting translation string:

```
od_waiting = "{holds, plural, =1 {1 person} other {# people}} waiting"
```

This gives more fine-grained control over pluralization support in languages with more complex requirements.